### PR TITLE
Integrate Spotify Audio Features

### PR DESCRIPTION
**Pull request by SunnyShikhar**
- Integrated Spotify Audio Features
- Tested on front end (but didn't change code in this PR for FrontEnd) to display audio metrics with `track_lyrics.audioFeatures().get('valence')`
- Must put client id and secret key in virtual environments `oauth2.py` for each user (if/when we deploy, Heroku will have a textbox for us to add it in so that we don't expose them)

Output in command line: 
image

### CODE REVIEW

**Comment by Janujan** [comment]
looks solid! I wasn't able to verify the stats because i didn't put down my client id/password.

**Comment by SunnyShikhar** [comment]
> looks solid! I wasn't able to verify the stats because i didn't put down my client id/password.

Yeah I'm wondering best way to setup for deployment. I'm fairly confident that since we have it in our `requirements.txt` when Heroku or w.e. installs it on it's instances, we can config a client and secret key then. 

Another approach could be to put the spotipy directory in the root folder and not have it as a pip dependency?

File: lyricize/requirements.txt [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
 numpy==1.15.4
 pandas==0.23.4
+Pillow==5.4.1
+pyparsing==2.3.1
 python-dateutil==2.7.5
 pytz==2018.7
 requests==2.21.0
 singledispatch==3.4.0.3
 six==1.12.0
+spotipy==2.4.4
```
Comment (SunnyShikhar) [inline lyricize/requirements.txt]: I'm gonna remove all the new additions to this file except for this line, re-install the modules and see if it works tonight before merging. All the additional modules are from the wordcloud PR before 😞

**Comment by SunnyShikhar** [comment]
To Do:

- [ ] Remove unnecessary modules from `requirements.txt` being introduced in this PR
- [ ] Change `Song_Lyrics` class to `Song` as the class now tracks title, artist and lyrics
- [ ] Consider storing metrics in `self.audioFeatures` and `self.lexicalRichness` for use in other functions than the ones calculating it

**Comment by SunnyShikhar** [comment]
Going to squash and merge the PR as it is - will worry about storing Spotify attributes within the class if the need arises for it.

**Pull request merged**